### PR TITLE
Padding fixed

### PR DESCRIPTION
**Pull request by prairir**
Added padding by switching from dvipng to imagemagick convert.
Imagemagick convert is much more flexible making it much easier to add large changes in the future. Not to mention it adds a large level of variability to the options that are already in use. 

I also added a box to the latex as suggested and done by @ElliotZ

### CODE REVIEW

**Comment by ElliotZ** [comment]
yeet

File: services/latex_service.rb [hunk 1]
```diff
@@ -17,8 +17,8 @@ def self.render?(message, path, file)
     # -T is image size
     # -o is output file
     if did_comp
-      system("dvipng -q* -D 300 -T tight #{File.join(path, file)}.dvi -o #{File.join(path, file)}.png")
-    else 
+      system("convert -density 300 -flatten #{File.join(path, file)}.dvi #{File.join(path, file)}.png >>/dev/null")
+    else
       return false
     end
```
Comment (EricPickup) [inline services/latex_service.rb]: Not sure why GitHub isn't letting me make a commit suggestion, but normally you can just reduce a statement like this to this:
```
return false unless did_comp
system("convert -density 300 -flatten #{File.join(path, file)}.dvi #{File.join(path, file)}.png >>/dev/null")
```

**Review by EricPickup — CHANGES_REQUESTED** [review]
Left one comment but otherwise LGTM, thanks @ElliotZ and @prairir 🎉

**Comment by prairir** [comment]
I made the changes and so il close this pull request.

**Pull request closed without merging**